dmapiclient/base.py

```diff
@@ -76,6 +76,9 @@ def _delete_with_updated_by(self, url, data, user):
     def _request(self, method, url, data=None, params=None):
         if not self.enabled:
             return None
+        if self.base_url is None:
+            logger.info("{} has no URL configured".format(self.__class__.__name__))
```

Comment:
Okay, in which case for master can we make it raise instead of log? Just because the error message that results from a missing API config isn't very clear at the moment.